sympy/physics/mechanics/kane.py

```diff
@@ -697,10 +696,12 @@ def kanes_equations(self, FL, BL):
         Parameters
         ==========
 
-        FL : list
+        loads : list
             Takes in a list of (Point, Vector) or (ReferenceFrame, Vector)
```

Comment:
This should not reference "list". It should say something like: "An iterable of ..."